lib/API/Values/RemoteMediaItem.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\RemoteMedia\API\Values;
+
+use Doctrine\ORM\Mapping as ORM;
+use Netgen\RemoteMedia\API\Values\RemoteResourceLocation;
+use Netgen\RemoteMedia\API\Values\TimestampableTrait;
+
+/**
+ * @ORM\Entity
```

Comment:
Bad annotations! :)

Please migrate Doctrine config to XML.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-remote-media/pull/15/commits/bf5c334bf66ff0cb669d85f62174d003c17912df.